porting/build_and_boot/H9_build.rst

```diff
@@ -63,6 +63,28 @@ If your makefile already includes a line beginning with ``BOARD_KERNEL_CMDLINE``
 
 .. Note::
     In rare cases the bootloader overwrites the kernel command line argument, rendering the setting above useless. This is the case for the Google Pixel 3a (sargo). To deal with this issue, replicate `this commit <https://github.com/fredldotme/android_kernel_google_bonito/commit/d0741dded3907f2cf4ecdc02bfcb74fc252763ff>`_. 
+    
+Setting system as root
+^^^^^^^^^^^^^^^^^^^^^^
+
+For halium 9, we need the system image to be built as system-as-root (for treble devices it should already be there ). Please check your ''boardconfig.mk'' for the following line, if it's there you can skip this step::
```

Comment:
halium 9 -> Halium 9

"there )" -> "there)" remove extra space

formatting of boardconfig should be with two backticks not two single quotes, ie ``boardconfig`` https://sphinx-tutorial.readthedocs.io/step-1/#code-samples